source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -320,6 +330,7 @@ class WordDocument(UIADocumentWithTableNavigation,WordDocumentNode,WordDocumentB
 	# Microsoft Word duplicates the full title of the document on this control, which is redundant as it appears in the title of the app itself.
 	name=u""
 
+	@script(gesture ="kb:NVDA+alt+c")

```

Comment:
This lacks the description. Please reuse the description for the script for the object model implementation of Word.

```suggestion
	@script(
		gesture="kb:NVDA+alt+c",
		# Translators: a description for a script that reports the comment at the caret.
		description=_("Reports the text of the comment where the System caret is located.")
	)
```